### PR TITLE
Update the Skills course for Actions

### PR DESCRIPTION
**Pull request by hubwriter**
### Summary

I worked through the Skills course for Actions and made a few changes while I was doing this.

### Changes

Changes to clarify some instructions and add a few missing steps.

First part of: https://github.com/github/docs-content/issues/14220

### Previews

- [README.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/README.md)
- [.github/steps/1-create-a-workflow.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/.github/steps/1-create-a-workflow.md)
- [.github/steps/2-add-a-job.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/.github/steps/2-add-a-job.md)
- [.github/steps/3-add-actions.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/.github/steps/3-add-actions.md)
- [.github/steps/4-merge-your-pull-request.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/.github/steps/4-merge-your-pull-request.md)
- [.github/steps/5-trigger.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/.github/steps/5-trigger.md)
- [.github/steps/X-finish.md](https://github.com/skills/hello-github-actions/blob/hubwriter/actions-skills-course/.github/steps/X-finish.md)

### Task list

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: .github/steps/1-create-a-workflow.md [hunk 1]
```diff
@@ -20,24 +12,30 @@ _Welcome to "Hello GitHub Actions"! :wave:_
...
        types: [opened]
    permissions:
      pull-requests: write
    ```
-1. To commit your changes, click **Commit new file**.
-1. Wait about 20 seconds for actions to run, then refresh this page (the one you're following instructions from) and an action will automatically close this step and open the next one.
+
+1. To commit your changes, click **Commit changes**.
+1. Type a commit message, select **Commit directly to the welcome-workflow branch** and click **Commit changes**.
+1. Wait about 20 seconds, then refresh this page (the one you're following instructions from). A separate Actions workflow in the repository (not the workflow you created) will run and will automatically replace this content with instructions for the next step.
```
Comment (felicitymay) [inline .github/steps/1-create-a-workflow.md:41]: I'm wondering if we should mention that this is the `README` file?

File: .github/steps/3-add-actions.md [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
 
 _Nice work adding a job to your workflow! :dancer:_
 
-Workflows have jobs, and jobs have steps. So now we'll add steps to your workflow.
+Workflows have jobs, and jobs have steps. So now we'll add a step to your workflow.
+
+**What are _steps_?**: Actions steps run - in the order they are specified, from the top down - when a workflow job is processed. Each step must pass for the next step to run.
 
-**What are _steps_?**: Actions steps will run during our job in order. Each step is either a shell script that will be executed, or an action that will be run. Each step must pass for the next step to run. Actions steps can be used from within the same repository, from any other public repository, or from a published Docker container image.
+Each step consists of either a shell script that's executed, or a reference to an action that's run. When we talk about an action (with a lowercase a) in this context, we mean a reusable unit of code. You can find out about actions in "[Finding and customizing actions](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions)," but for now we'll use a shell script in our workflow step.
```
Comment (felicitymay) [inline .github/steps/3-add-actions.md:9]: Love this additional explanation, but maybe:

```suggestion
Each step consists of either a shell script that's executed, or a reference to an action that's run. When we talk about an action (with a lowercase "a") in this context, we mean a reusable unit of code. You can find out about actions in "[Finding and customizing actions](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions)," but for now we'll use a shell script in our workflow step.
```

File: .github/steps/1-create-a-workflow.md (hunk 1 above)
Comment (hubwriter) [inline .github/steps/1-create-a-workflow.md:41]:
```suggestion
1. Wait about 20 seconds, then refresh this page (the one you're following instructions from). A separate Actions workflow in the repository (not the workflow you created) will run and will automatically replace this contents of this README file with instructions for the next step.
```

Comment (sarobakhtiyari) [reply .github/steps/1-create-a-workflow.md]: []()

**Review by Axelmj2006 — COMMENTED** [review]
@felicitymay [](url)

**Comment by Axelmj2006** [comment]
@Axelmj2006

**Pull request merged**